yfaq/references_to_mbc.md

```diff
@@ -4,7 +4,21 @@ The following table contains community references to MBC:
 
```

Comment:
This sentence should be updated to, "The following table lists papers and presentations that reference MBC."